### DISCUSSION TITLE
Is package downgrade necessary

### DISCUSSION THREAD

**Discussion by thejuskambi** [pythonnet/pythonnet#2165] [Q&A]
While trying to install pythonnet, I  got message that it would downgrade python and openssl.

> The following packages will be DOWNGRADED:
>  openssl                                          3.0.7-0 --> 1.1.1n-0
>  python                                           3.9.16-0 --> 3.9.11-2

I am using `conda install pythonet`. 

I was wondering why should pythonnet be so strict about dependencies?

**Comment by filmor** [comment]
This can't have anything to do with Python.NET, we (and also the conda-forge package) place no restrictions on individual Python versions or openssl.
